client/state/sharing/schema.js

```diff
@@ -0,0 +1,19 @@
+import {
```

Comment:
I thought about this a bit, and re-added this, and made sure to wire it up this time. I'll update the PR with examples of inconsistent state.